Update Pomerium reverse proxy guide (#7078)
jenkins-infra/jenkins.io: content/doc/book/system-administration/reverse-proxy-configuration-with-jenkins/reverse-proxy-configuration-pomerium.adoc

|`email`

|**Acceptable issuers**
|`authenticate.corp.example.com`

|**Acceptable audiences**
|`jenkins.corp.example.com`

|**JWKS JSON URL**
|`https://jenkins.corp.example.com/.well-known/pomerium/jwks.json`
|===

Note the following details about the fields above:

- **Username claim name** can be either your name or email
- **Acceptable issuers** must be the URL of the authentication domain that issued the JWT. The `iss` claim tells the target application who the issuing authority is and provides context about the subject.
- **Acceptable audiences** must be the URL of the target application. The `aud` claim defines what application the JWT is intended for.
- **JWKS JSON URL** appends `/.well-known/pomerium/jwks.json` to the external route URL. The JWKS endpoint provides Jenkins the user’s public key to verify their JWT signature.

You can go to the external `verify` route defined in your policy to view your JWT claims.

In the **Authorization** dropdown, configure Jenkins permissions so that **Anonymous** has **Administer** privileges.

1. Select **Matrix-based security**
2. Under **Overall**, assign **Administer** to **Anonymous** and **Authenticated Users**

> If JWT authentication doesn't authenticate you successfully, Jenkins will sign you in as an anonymous user. With administer privileges, you can troubleshoot JWT settings as an anonymous user and try again.

Reviewer: ```suggestion
If JWT authentication doesn't authenticate you successfully, Jenkins signs you in as an anonymous user. With administer privileges, you can troubleshoot JWT settings as an anonymous user and try again.
```
Suggestion to use [active voice](https://learn.microsoft.com/en-us/style-guide/global-communications/writing-tips#tips-for-all-global-content) here instead of "will sign".

Select **save** to apply the security settings.

=== Test JWT authentication

> Restart your container. If the JWT authentication worked, you will see your name in the dashboard instead of **admin**. To see more details about the request, add `/whoAmI` to the URL. For example, `https://jenkins.localhost.pomerium.io/whoAmI`.

Reviewer: ```suggestion
Restart your container. If the JWT authentication worked, your name appears in the dashboard instead of **admin**. To see more details about the request, add `/whoAmI` to the URL. For example, `https://jenkins.localhost.pomerium.io/whoAmI`.
```
Suggestion to use [active voice](https://learn.microsoft.com/en-us/style-guide/global-communications/writing-tips#tips-for-all-global-content) here instead of "will see".
